src/post/Write/PostEditor.scss

```diff
@@ -90,15 +98,118 @@
     user-select: none;
   }
 
+  $iconPaddingY: 4px;
+  $iconPaddingX: 2px;
   .PostEditor__styleButton {
-    color: #999;
-    cursor: pointer;
-    margin-right: 16px;
-    padding: 2px 0;
+    margin: 0;
+    padding: $iconPaddingY $iconPaddingX;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
     display: inline-block;
+    cursor: pointer;
+
+    i{
+      color: #999;
+      &:hover{
+        color:#fff;
+      }
+    }
   }
 
   .PostEditor__activeButton {
-    color: #5890ff;
+    i{
+      color: #5890ff;
+    }
+  }
+  
+  $toolbarBackground:#000;
+  .NewPost__toolbar{
+    background: $toolbarBackground;
+    position: absolute;
+    display: none;
+    z-index: 10;
+
+    .toolbar__button{
+      background: none;
+      border: none;
+      outline: none;
+    }
+
+    &__visible{
+      display: block;
+    }
+
+    &__wrapper {
+      background-color: $toolbarBackground;
+      border-radius: 4px;
+      box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.4);
+      left: -50%;
+      position: relative;
+      transition: background-color 0.2s ease-in-out;
+    }
+
+    .toolbar__list{
+      padding: 0 8px;
+      margin: 0;
+      white-space: nowrap;
+      list-style: none;
+
+      &::after{
+        content: '';
+        display: inline-block;
+        top: 100%;
+        left: 50%;
+        position: absolute;
+        pointer-events: none;
+        width: 0;
+        height: 0;
+        margin-left: -8px;
+        border-width: 8px;
+        border-style: solid;
+        border-color: $toolbarBackground transparent transparent;
+      }
+    }
+    
+    .toolbar__item__separator{
+      list-style: none;
+      border-right: 1px solid #ddd;
+      margin-right: $iconPaddingX;
+      height: 1.5em;
+      vertical-align: middle;
+    }
+  }
+
+  .SideControls{
+    display: block;
+    position: absolute;
+    z-index: 2;
+
+    button{
```

Comment:
It's much better to avoid styling html elements in all cases. If it's possible use a classname to avoid style conflicts